src/femtomes.cpp

```diff
@@ -434,3 +639,101 @@ GPSDriverFemto::calculateBlockCRC32(uint32_t length, uint8_t *buffer, uint32_t c
 	return (crc);
 }
 
+void GPSDriverFemto::activateCorrectionOutput()
+{
+    if (_correction_output_activated || _output_mode != OutputMode::RTCM)
+    {
+        return;
+    }
+
+    char buffer[100];
+
+    if (_base_settings.type == BaseSettingsType::survey_in)
+    {
+        FEMTO_DEBUG("Femto: enabling survey-in")
+
+        if(writeAckedCommandFemto("POSAVE ON \r\n","<POSAVE OK",FEMTO_RESPONSE_TIMEOUT) == 0)
+        {
+            FEMTO_DEBUG("Femto: command POSAVE ON success")
+
+            if (writeAckedCommandFemto("LOG GPGGA 1 \r\n", "<LOG OK",FEMTO_RESPONSE_TIMEOUT) == 0) /**< for updating GPS satellite count of RTK */
+            {
+                FEMTO_DEBUG("Femto: command LOG GPGGA 1 success")
+            }
+            else
+            {
+                FEMTO_ERR("Femto: LOG GPGGA command failed")
+            }
+        }
+        else
+        {
+            FEMTO_ERR("Femto: command POSAVE ON failed")
+        }
+
+        _base_settings.settings.survey_in.min_dur = 0; // use it as counter how long survey-in has been active
```

Comment:
Where is this incremented? It should be updated once per second, which is displayed in QGC

Reply:
Thanks for reminding! I forgot to implement this. I will implement it later.